src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -37,13 +36,12 @@ public class LogicManager implements Logic {
     private final Model model;
     private final Storage storage;
     private final AddressBookParser addressBookParser;
-    private final CommandHistory commandHistory;
 
     /**
      * Constructs a {@code LogicManager} with the given {@code Model} and {@code Storage}.
      */
     public LogicManager(Model model, Storage storage) {
-        this.commandHistory = new CommandHistory();
+        //this.commandHistory = new CommandHistory();
```

Comment:
Think we can delete this comment since its redundant now?

Reply:
My bad for missing that out, I have removed it